docs/ad/index.md

```diff
@@ -40,36 +45,54 @@ A feature is the field in your index that you want to find anomalies for. A dete
+So, if the detector interval is 10 min, the detector might take more than an hour to initialize.
+If you see the detector pending in initialization for a long time, increase the detector interval to fasten the process.
+Or, use the [profile detector](./api#profile-detector) operation to make sure you have sufficient data in a shingle size of 8.
+
+![Anomaly detection results](../images/ad.png)
+
+The **Live anomalies** chart shows you the live anomaly results for the last 60 intervals. For example, if the interval is set to 10, it shows the results for the last 600 minutes.
+
+The **Anomaly history** chart plots the anomaly grade with the corresponding measure of confidence.
 
 Anomaly grade is a number between 0 and 1 that indicates the level of severity of how anomalous a data point is. An anomaly grade of 0 represents “not an anomaly,” and a non-zero value represents the relative severity of the anomaly. The confidence score is an estimate of the probability that the reported anomaly grade matches the expected anomaly grade. Confidence increases as the model observes more data and learns the data behavior and trends. Note that confidence is distinct from model accuracy.
 
-The **Features** graph plots the features based on the aggregation method. On the top-right corner, you can vary the date-time range of the detector.
+The **Feature breakdown** graph plots the features based on the aggregation method. On the top-right corner, you can vary the date-time range of the detector.
+
+### Step 4: Set up alerts
 
-To see all the anomalies that are detected, choose the **Anomalies** tab.
+To create a monitor to send you notifications when any anomalies are detected, choose **Set up alerts**.
+You're redirected to the **Alerting**, **Add monitor** page.
 
-### Step 4: Adjust the model
+For steps to create a monitor and set notifications based on your anomaly detector, see [Monitor](../alerting/monitors/).
 
-You can enable or disable features or fine tune the time interval to minimize any false positives.
+If you stop or delete a detector, make sure to delete any monitors associated with the detector, as this is not done automatically.
 
-1. Choose **Adjust Model**.
-2. In **Global Settings**, specify the time interval in minutes.
+### Step 5: Adjust the model
 
-### Step 5: Create a monitor
+To see all the configuration settings, choose the **Detector configuration** tab.
 
-To create a monitor to send you notifications when any anomalies are detected, choose **Actions** and select **Create Monitor**. For steps to create a monitor and set notifications based on your anomaly detector, see [Monitor](../alerting/monitors/).
+1. To make any changes to the detector configuration, in the **Detector configuration** section, choose **Edit**
+1. To enable or disable features or fine tune the time interval to minimize any false positives, in the **Features** section, choose **Edit**.
```

Comment:
Time interval is on editing detector configuration page, not editing feature page.

Reply:
Changed.